### ISSUE TITLE
LLaMA instalation getting stuck after about 30 min

### ISSUE THREAD

**Issue by uwuclxdy** [cocktailpeanut/dalai#267]
I can't get past consolidated.01.pth download where it won't do anything. It just says ETA 14 min but it's not moving / downloading at all. This is my 5th try and I don't know what to do.

Windows 10, AMD machine, everything installed properly, running in cmd.

**Comment by 1larity** [comment]
I don't think this is an issue with this project, just with the server providing the download, which is a third party I believe. For the larger models I had to retry several times to get them to download. just hit CTRL-C and try again.

**Comment by antesunkis** [comment]
I also have the same problem. I would say it's kind of related to the project when you have to download the same 16G files over and over again just because it gets stuck at one point or another.

**Comment by watarungurunnn** [comment]
I downloaded the weights directly from https://agi.gpt4.org/llama/LLaMA/[model]/consolidated.0[num].pth 
for example, https://agi.gpt4.org/llama/LLaMA/7B/consolidated.00.pth.
put it in the dalai/llama/models/7B dir and then run the command provided.

**Comment by dsmih** [comment]
Thank you, this fixed 7b for me, but with the others, I can't access to download the weights directly

https://agi.gpt4.org/llama/LLaMA/7B/consolidated.01.pth is not allowing download. Where can I download the 01?

**Comment by watarungurunnn** [comment]
It's because 7B needs only one pth file.

**Comment by dsmih** [comment]
> It's because 7B needs only one pth file.

Sorry I was exhausted when I posted that comment. Currently, I'm struggling to install the 13B as it keeps getting stuck trying to download the consolidated.01.pth file. I was hoping someone would be able to point me in the direction of downloading it directly as the links I found by myself don't seem to allow downloads.

**Comment by watarungurunnn** [comment]
How to download on Mac:
https://github.com/facebookresearch/llama/issues/53#issue-1606582963

**Comment by watarungurunnn** [comment]
@dsmih 
https://agi.gpt4.org/llama/LLaMA/13B/consolidated.01.pth
